docs/gettingstarted/index.rst

```diff
@@ -105,10 +105,10 @@ Getting started with Flocker
 
          .. version-code-block:: console
 
-            you@laptop:~$ sudo yum install -y @buildsys-build python python-devel python-virtualenv && \
-              virtualenv flocker-tutorial && \
-              flocker-tutorial/bin/pip install --upgrade pip && \
-              flocker-tutorial/bin/pip install --quiet https://clusterhq-archive.s3.amazonaws.com/python/Flocker-|latest-installable|-py2-none-any.whl && source flocker-tutorial/bin/activate
+            you@laptop:~$ sudo yum install -y @buildsys-build python python-devel python-virtualenv libffi-devel openssl-devel && \
+              curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/linux-install.sh && \
+              sh linux-install.sh && \
```

Comment:
Cool kids execute straight from the 'net, `curl https://.../linux-install.sh | sh`